apps/extension/src/shared/ui/components/icon-button/icon-button.component.tsx

```diff
@@ -32,3 +37,33 @@ export const IconButton = ({
     </button>
   );
 };
+
+export const IdrissIconButton = ({
```

Comment:
Why do we need to duplicate the IconButton?

Reply:
We can use the IconButton from the `@idriss-xyz/ui/icon-button`. That will be easier

Reply:
I have double checked, in the app we have two icon systems so it is not a mistake, I just used the other one

Reply:
Why can't we use `IconButton` from `packages/ui`?